.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+
+defaults: &defaults
+    working_directory: ~/bs-react-navigation
+    docker:
+        - image: circleci/node:9
+
+jobs:
+    install-dependencies:
+        <<: *defaults
+        steps:
+            - checkout
+            - attach_workspace:
+                at: ~/bs-react-navigation
+            - restore_cache:
+                keys:
+                - v1-24-dependencies-{{ checksum "package.json" }}
```

Comment:
Generate checksum `from yarn.lock` because when script change without dependencies we loose the cache :P